setup.py

```diff
@@ -44,3 +43,9 @@ def get_requirements():
     install_requires=get_requirements(),
     python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, <4',
 )
+
+from xanthos.install_supplement import InstallSupplement
+
+# install supplemental and example data from Zenodo
+print("Installing supplemental data from Zenodo...")
+InstallSupplement().fetch_zenodo()
```

Comment:
If you don't mind, I'd like to add a newline to the end of this file.
```suggestion
InstallSupplement().fetch_zenodo()

```

Reply:
Good for me. I meant to as well.